ext/gmp/config.m4

```diff
@@ -3,8 +3,11 @@ PHP_ARG_WITH(gmp, for GNU MP support,
 
 if test "$PHP_GMP" != "no"; then
 
+  MACHINE_INCLUDES=$($CC -dumpmachine)
```

Comment:
I've tried it in containers with ArchLinux and Fedora and they both compiled. You'd have to get more people to test it for the systems I can't think of or have reasonable access to.